deploy/crds/infra.watch_servicetelemetrys_crd.yaml

```diff
@@ -167,49 +167,71 @@ spec:
                     description: Events related backend configuration.
                     properties:
                       elasticsearch:
-                        description: Events storage backend ElasticSearch
+                        description: Events storage backend Elasticsearch
                         properties:
                           enabled:
-                            description: Enable ElasticSearch as a storage backend for events
+                            description: Enable Elasticsearch as a storage backend for events
                             type: boolean
+                          forwarding:
+                            description: Configuration for where to forward events
+                            type: object
+                            properties:
+                              hostUrl:
+                                description: URL of Elasticsearch HTTP(S) endpoint
+                                type: string
+                              tlsServerName:
+                                description: (if required) Server Name expected to match the certificate presented by the endpoint
+                                type: string
+                              tlsSecretName:
+                                description: (if required) Name of the secret that stores the CA cert and client cert/key
+                                type: string
+                              userSecretName:
+                                description: (if required) Name of the secret that stores the Basic Auth credentials
+                                type: string
+                              useBasicAuth:
+                                description: Whether to provide HTTP Basic Auth headers
+                                type: boolean
+                              useTls:
+                                description: Whether to enable TLS
+                                type: boolean
                           version:
-                            description: Version of ElasticSearch to deploy. Elasticsearch licensing has changed as of version 7.11. See https://www.elastic.co/pricing/faq/licensing for details.
+                            description: (DEPRECATED - Use ext* after STF 1.5.3) Version of Elasticsearch to deploy. Elasticsearch licensing has changed as of version 7.11. See https://www.elastic.co/pricing/faq/licensing for details.
```

Comment:
I think all the descriptions need to be updated here. They refer to `ext` prefix which you recently removed.

Reply:
Good catch, I'll send a patch soon